{{cookiecutter.project_shortname}}/docker-compose.full.yml

```diff
@@ -15,7 +15,7 @@
 # - Cache: Redis (exposed port: 6379)
 # - DB: (PostgresSQL/MySQL) (exposed port: 5432 or 3306)
 # - Message queue: RabbitMQ (exposed ports: 5672, 15672)
-# - Search platform: (OpenSearch/Elasticsearch) (exposed ports: 9200, 9300)
+# - Search platform: (OpenSearch) (exposed ports: 9200)
```

Comment:
```suggestion
# - Search platform: (OpenSearch) (exposed ports: 9200, 9600)
```
The performance analyzer runs there and is exposed: https://github.com/inveniosoftware/cookiecutter-invenio-rdm/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/docker-services.yml#L114